TKGame/Components/Concrete/Player_PhysicsComponent.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.Components.Interface;
+
+namespace TKGame.Components.Concrete
+{
+    internal class Player_PhysicsComponent : PhysicsComponent
+    {
+        private static readonly float GRAVITY = 1.0f;
```

Comment:
We may want to adjust the gravity value in the future to actually keep the entities on the platform most of the time. Right now they can just run up vertically, overcoming gravity. However, this is fine for now.

Reply:
Potentially, yes, but that's only if we give them the ability to simply move vertically without jumping, which I don't think is the plan